### ISSUE TITLE
Support an undelete/restore option

### ISSUE THREAD

**Issue by rkaippully** [rkaippully/gamgee#3]
It will be a good idea to support an undelete/restore option after a token is deleted.

**Comment by vmoti** [comment]
Do you mean a token would be merely soft-deleted, marked for proper deletion after 30 days? If that's what you had in mind, are you sure that's a good idea from a security perspective?

**Comment by rkaippully** [comment]
Yes, this is to help in cases where you accidentally delete a token. There can always be a "force" delete option to completely remove it if that is desired.
